pkg/bootstrap/testdata/all_golden.json

```diff
@@ -342,7 +342,7 @@
           "name": "envoy.transport_sockets.tls",
           "typed_config": {
             "@type": "type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext",
-            "sni": "",
+            "sni": "mypilot",
```

Comment:
is this just the test doing something odd? Or will normal bootstrapactually be changed?

Reply:
that's the effect of  `discHost` computation line that got moved. it is now exercised in this test, but was passed as "" before.